server/chat-plugins/helptickets.ts

```diff
@@ -793,7 +793,7 @@ export async function getBattleLog(battle: string): Promise<BattleInfo | null> {
 				url: `https://${Config.routes.replays}/${battle}`,
 			};
 		}
-	} catch (e) {}
+	} catch (e: any) {}
```

Comment:
Or this?

Reply:
https://github.com/smogon/pokemon-showdown/pull/8449#discussion_r698140489